{"name": "test_2_CreateLog", "status": "broken", "statusDetails": {"message": "TypeError: 'tuple' object is not callable", "trace": "self = <testcases.test_create_log.TestCreate_log object at 0x03C2C700>\ntest_logsystem = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"b10ebb7c-f818-4eec-9bdb-79fc3c11fca7\")>\n\n    def test_2_CreateLog(self,test_logsystem):\n        self.page1 = TestLogPage(test_logsystem)\n        self.page1.type_blog_title(\"我最帅\")\n        self.page1.type_tag(\"个人属性\")\n        self.page1.type_ke_content(\"我最帅，在座的各位谁赞成，谁反对？\")\n        time.sleep(2)\n>       self.page1.select_album()\nE       TypeError: 'tuple' object is not callable\n\ntest_create_log.py:33: TypeError"}, "start": 1628836194809, "stop": 1628836196861, "uuid": "aabe5d43-254c-4f6f-9570-9544ae2f57c3", "historyId": "4cc768d0db640219c705049bf0abf532", "testCaseId": "b1dec58d31e6c5197e5bdc5714d1b0e5", "fullName": "testcases.test_create_log.TestCreate_log#test_2_CreateLog", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_create_log"}, {"name": "subSuite", "value": "TestCreate_log"}, {"name": "host", "value": "DESKTOP-S2QKEKE"}, {"name": "thread", "value": "7336-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_create_log"}]}